src/test/regress/expected/create_drop_database_propagation.out

```diff
@@ -0,0 +1,260 @@
+ [{"datacl": null, "datname": "mydatabase", "datctype": "C", "encoding": "UTF8", "datcollate": "C", "tablespace": "create_drop_db_tablespace", "datallowconn": true, "datconnlimit": 10, "datistemplate": false, "database_owner": "create_drop_db_test_user"}]
+(3 rows)
+
+drop database mydatabase;
+SELECT result from run_command_on_all_nodes(
+  $$
+  SELECT jsonb_agg(to_jsonb(q2.*)) FROM (
+    SELECT pd.datname, pg_encoding_to_char(pd.encoding) as encoding,
+    pd.datistemplate, pd.datallowconn, pd.datconnlimit,
+    pd.datcollate , pd. datctype  ,  pd.datacl,
+    pa.rolname AS database_owner, pt.spcname AS tablespace
+    FROM pg_database pd
+    JOIN pg_authid pa ON pd.datdba = pa.oid
+    join pg_tablespace pt on pd.dattablespace = pt.oid
+    WHERE datname = 'mydatabase'
+  ) q2
+  $$
+) ORDER BY result;
+ result
+---------------------------------------------------------------------
+
+
+
+(3 rows)
+
+-- test database syncing after node addition
+select 1 from citus_remove_node('localhost', :worker_2_port);
+ ?column?
+---------------------------------------------------------------------
+        1
+(1 row)
+
+--test with is_template true and allow connections false
+CREATE DATABASE mydatabase
+            OWNER = create_drop_db_test_user
+            CONNECTION LIMIT = 10
+            ENCODING = 'UTF8'
+            TABLESPACE = create_drop_db_tablespace
+            ALLOW_CONNECTIONS = false
+            IS_TEMPLATE = false;
+SELECT result from run_command_on_all_nodes(
+  $$
+  SELECT jsonb_agg(to_jsonb(q2.*)) FROM (
+    SELECT pd.datname, pg_encoding_to_char(pd.encoding) as encoding,
+    pd.datistemplate, pd.datallowconn, pd.datconnlimit,
+    pd.datcollate , pd. datctype  ,  pd.datacl,
+    pa.rolname AS database_owner, pt.spcname AS tablespace
+    FROM pg_database pd
+    JOIN pg_authid pa ON pd.datdba = pa.oid
+    join pg_tablespace pt on pd.dattablespace = pt.oid
+    WHERE datname = 'mydatabase'
+  ) q2
+  $$
+) ORDER BY result;
+                                                                                                                             result
+---------------------------------------------------------------------
+ [{"datacl": null, "datname": "mydatabase", "datctype": "C", "encoding": "UTF8", "datcollate": "C", "tablespace": "create_drop_db_tablespace", "datallowconn": false, "datconnlimit": 10, "datistemplate": false, "database_owner": "create_drop_db_test_user"}]
+ [{"datacl": null, "datname": "mydatabase", "datctype": "C", "encoding": "UTF8", "datcollate": "C", "tablespace": "create_drop_db_tablespace", "datallowconn": false, "datconnlimit": 10, "datistemplate": false, "database_owner": "create_drop_db_test_user"}]
+(2 rows)
+
+select 1 from citus_add_node('localhost', :worker_2_port);
```

Comment:
I'd like to see the the CREATE DATABASE command that is being sent here. Can you add:
```
SET citus.log_remote_commands = true;
set citus.grep_remote_commands = '%CREATE DATABASE%';
```

Reply:
Fixed